Source/WebCore/loader/cache/CachedFontLoadRequest.h

```diff
@@ -60,7 +60,16 @@ class CachedFontLoadRequest final : public FontLoadRequest, public CachedFontCli
     bool isLoading() const final { return m_font->isLoading(); }
     bool errorOccurred() const final { return m_font->errorOccurred(); }
 
-    bool ensureCustomFontData() final { return m_font->ensureCustomFontData(); }
+    bool ensureCustomFontData() final
+    {
+        bool result = m_font->ensureCustomFontData();
+        if (!result && m_font->didRefuseToParseCustomFont() && m_context) {
+            auto message = makeString("[Lockdown Mode] This font has been blocked: ", m_font->url().string());
+            m_context->addConsoleMessage(MessageSource::Security, MessageLevel::Info, message);
+        }
```

Comment:
Nit: This is not new with your patch, but all of this is a lot of complexity to put in a header file. We should make a `CachedFontLoadRequest.cpp` file to hold this kind of stuff.